Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,34 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            string name = "Tom Jefferson";
```

Comment:
Avoid delcaring variables at the top. Declare them as close as possible to where they are used.